### PR TITLE
new block for bildwurf ads

### PR DESCRIPTION
**Pull request by roosnic1**
WPC-530. This PR adds a new embed block for bildwurf ads.

### CODE REVIEW

**Review by bishoymelek — DISMISSED** [review]
🎉

File: examples/website/src/shared/blocks/embedBlock.tsx [hunk 1]
```diff
@@ -34,7 +35,7 @@ const EmbedBlockStyle = cssRule<{showBackground: boolean}>(({showBackground}) =>
   }),
 
   ...whenDesktop({
-    width: '50%',
+    //width: '50%',
```
Comment (bishoymelek) [inline examples/website/src/shared/blocks/embedBlock.tsx:38]: Do you wanna clean this? 👀

File: examples/website/src/shared/atoms/bildwurfAdEmbed.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    () => !!window._ASO,
+    false
+  )
+
+  useEffect(() => {
+    load()
+    try {
+      window._ASO.loadAd('bildwurf-injection-wrapper', zoneID)
+    } catch (error) {
+      console.warn('could not call _ASO.loadAd()')
```
Comment (bishoymelek) [inline examples/website/src/shared/atoms/bildwurfAdEmbed.tsx:28]: Do you want to specify that it's related to `bildwurf`?

**Review by camilleory — APPROVED** [review]
Are Bildwurf going to provide their adds in an embed format (and if yes do we have examples)?

File: examples/website/src/shared/atoms/bildwurfAdEmbed.tsx (hunk 2 above)
Comment (roosnic1) [reply examples/website/src/shared/atoms/bildwurfAdEmbed.tsx:28]: `_ASO` === Bildwurf

**Review by penk85 — APPROVED** [review]
:+1:

**Review by bishoymelek — DISMISSED** [review]
🎉

**Review by camilleory — DISMISSED** [review]

**Review by penk85 — APPROVED** [review]

**Review by camilleory — APPROVED** [review]

**Pull request merged**